_posts/2015-10-29-code-talks-java.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "code.talks: code & nachos"
+date: "2015-10-29 09:00:00"
+icon: comments
+categories: conferences
+authors: ["Erik"]
+---
+
+Some weeks ago we attended the `code.talks` developer conference in Hamburg.
+Besides the amazing location (which was the Cinemaxx Dammtor) and the awesome catering (nachos, popcorn and Mate as much as you can endure) the conference was also (who would have guessed that) packed with lots of interesting talks and people.
+
+In the variety of talks we found things we're already doing right (according to the speakers) and other aspects we can improve on.
+There were also talks we could contribute to since we have solved their problems already.
+I'd like to cover **three** points that I realised during this conference in the following paragraphs.
```

Comment:
I'd like to cover **three** main points that I got during this conference in the following paragraphs.